content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -39,17 +39,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/instance=mysql-abcxzy`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
 
-A unique name identifying the instance of an application.
+A unique name identifying the instance of an application. If you want to declare what application this is part of but not in a unique way, use [app.kubernetes.io/name](##app-kubernetes-io-name).
```

Comment:
Use [simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language), Markdown nit

```suggestion
A unique name identifying the instance of an application. To assign a non-unique name, use [app.kubernetes.io/name](#app-kubernetes-io-name).
```